packages/transpiler/src/transformations/index.ts

```diff
@@ -0,0 +1,19 @@
+import { appendDirective } from './append-directive';
```

Comment:
We can use `export ... from` in this file.

```suggestion
export { appendDirective } from './append-directive';
```